lib/styles/models/box.less

```diff
@@ -1,8 +1,6 @@
 @prefix: graphjs;
 
 .@{prefix}-box.@{prefix}-root {
-    overflow-x: hidden;
-    overflow-y: auto;
```

Comment:
why would he put these?

Reply:
don't forget, certain elements you believe they are unnecessary, but they are there to "reset". If they are not there, they mix up with parent CSS values, hence render bad on some sites.